grails-app/services/org/pih/warehouse/api/MegamenuService.groovy

```diff
@@ -50,6 +50,10 @@ class MegamenuService {
     List buildAndTranslateSubsections(List subsections, User user, Location location) {
         def builtSubsections = []
         subsections.each {
+            def minRole = it.minimumRequiredRole
+            if (minRole && !userService.isUserInRole(user, minRole)) {
+                return
+            }
             def roles = it.requiredRole
             if (roles && !userService.hasAnyRoles(user, roles)) {
```

Comment:
Is the `requiredRole` still used in the config?

Reply:
it is used only on invoicing sections, where user is required to have `ROLE_INVOICE`

```
        invoicing {
            enabled = true
            requiredRole = [RoleType.ROLE_INVOICE]
            label = "react.invoicing.label"
            defaultLabel = "Invoicing"
            subsections = [ ... ]
        }
```